src/script/components/userDevices/DeviceDetails.tsx

```diff
@@ -75,7 +75,7 @@ const DeviceDetails: React.FC<DeviceDetailsProps> = ({
   useEffect(() => {
     setFingerprintRemote(undefined);
     if (selectedClient) {
-      cryptographyRepository
+      void cryptographyRepository
```

Comment:
what is the point of void in this case? im not so familiar with it here

Reply:
is void the most appropriate handling in this case? we dont need to await the response?